apps/khix/src/app/credits/page.module.css

```
@@ -0,0 +1,749 @@
+    #000000 48%
+  );
+  mask-image: linear-gradient(
+    to bottom,
+    transparent 0%,
+    rgba(0, 0, 0, 0.12) 18%,
+    #000000 48%
+  );
+  pointer-events: none;
+}
+
+@keyframes copyEntrance {
```

Stylelint (17.14.0)

[error] Expected keyframe name "copyEntrance" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)